src/rlookup.c

```diff
@@ -417,43 +417,93 @@ int RLookup_GetHash(RLookup *it, RLookupRow *dst, RedisModuleCtx *ctx, RedisModu
   return rc;
 }
 
+// We can only use the scan API from Redis version 6.0.6 and above
+// because we need this fix: https://github.com/redis/redis/commit/51e178454d
+static inline int canUse_RM_ScanAPI() {
+  return !((redisVersion.majorVersion < 6) ||
+           (redisVersion.majorVersion == 6 && redisVersion.minorVersion == 0 && redisVersion.minorVersion <= 5)
+          );
+}
```

Comment:
Use `isFeatureSupported` instead.

Reply:
@ashtul as I wrote above, it's not that the feature is not supported, you can not use it before version 6.0.6 because of a bug:
https://github.com/RediSearch/RediSearch/pull/1879#issuecomment-786164833

Reply:
Ohh @ashtul got you:
```
static inline int isFeatureSupported(int feature) {
  return feature <= RSGlobalConfig.serverVersion;
}
```
I think we should change the name to something like `CheckRedisVersion` ....